<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoubleSlider</class>
 <widget class="QWidget" name="DoubleSlider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Label</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="minSlider">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="minValueLabel">
         <property name="text">
          <string>IV</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="maxSlider">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="maxValueLabel">
         <property name="text">
          <string>XV</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>maxSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>maxValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>minValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
